modules/labs-react/expandable/lib/ExpandableTarget.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+import {
+  createComponent,
+  ExtractProps,
+  focusRing,
+  hideMouseFocus,
+  styled,
+  StyledType,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+
+import {ExpandableModelContext} from './Expandable';
+import {DisclosureModel, DisclosureModelConfig} from '@workday/canvas-kit-react/disclosure';
+import {ExpandableTitle} from './ExpandableTitle';
+import {useExpandableTarget} from './hooks/useExpandableTarget';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {colors} from '@workday/canvas-kit-react/tokens';
+
+export interface ExpandableTargetProps
+  extends DisclosureModelConfig,
+    ExtractProps<typeof Flex, never> {
+  model?: DisclosureModel;
+  /**
+   * Children of the `Expandable.Target`. Should contain `Target.Title`, an optional `Target.Avatar`
+   * and either `Target.StartIcon` or `Target.EndIcon`. `Target.StartIcon` is meant to be placed before
+   * the `Target.Title` and `Target.EndIcon` should be placed after.
+   */
+  children: React.ReactNode;
+  /**
+   * Specifies the semantic heading level that will wrap the `Expandable.Target`'s button. If 'none' is
+   * selected the no semantic heading will wrap the button.
+   */
+  headingLevel: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'none';
```

Comment:
I added a `none` option so developers have the flexibility to implement their own heading.

```
<Expandable>
  <h3>
    <Expandable.Target headingLevel="none">
      <Expandable.StartIcon />
      <Expandable.Title>Title</Expandable.Title>
    </Expandable.Target>
  </h3>
  <Expandable.Content>Content</Expandable.Content>
</Expandable>
```